ansible/deploy_proxy.yml

```diff
@@ -66,7 +66,7 @@
           path: "{{ log_home }}/*.log"
           options:
             - monthly
-            - size 1G
+            - size 750M
```

Comment:
note this won't make a difference if more than 750 MB is written in a month. i think we should drop the check down to `daily` and lower the size further and just keep more compressed logs